editorconfig-eclipse-functional-test/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: EditorConfig Eclipse Functional Test
 Bundle-SymbolicName: editorconfig-eclipse-functional-test;singleton:=true
-Bundle-Version: 0.3.0.qualifier
+Bundle-Version: 0.4.0.qualifier
```

Comment:
I don't know if this needed to be changed, but it looked like here and in the POM the version wasn't bumped. I can remove this commit if it's not supposed to be here.